### PR TITLE
2.0.0-rtm to 2.1.0-preview1

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

**Comment by bricelam** [comment]
Search the codebase for `2.0.0`. There are a few other places that need to be updated.

**Comment by bricelam** [comment]
Also, we'll need to update the aspnet dependencies. Will that happen in a second phase?

**Comment by smitpatel** [comment]
Also change in `appveyor.yml` for book keeping.

**Comment by ryanbrandenburg** [comment]
The AspNet dependencies will be updated in a second pass once other repos have a change to have their packages be available.

File: test/EFCore.Design.Tests/Migrations/Design/SnapshotModelProcessorTest.cs [hunk 1]
```diff
@@ -126,7 +126,7 @@ public void Does_not_warn_for_duplicate_non_conflicting_annotations()
         public void Does_not_process_non_v1_models()
         {
             var model = new Model();
-            model.SetProductVersion("2.0.0");
+            model.SetProductVersion("2.1.0");
```
Comment (bricelam) [inline test/EFCore.Design.Tests/Migrations/Design/SnapshotModelProcessorTest.cs]: Can keep at 2.0.0

**Pull request merged**